### PR TITLE
Release 3.6.1

### PR DESCRIPTION
**Pull request by elgatovital**
@gnosis/mobile-devs

### CODE REVIEW

**Review by leobond2021 — APPROVED** [review]

**Review by leobond2021 — APPROVED** [review]

**Review by leobond2021 — APPROVED** [review]

File: app/proguard-rules.pro [hunk 1]
```diff
@@ -73,6 +73,9 @@
 -keep @interface pm.gnosis.heimdall.data.adapters.DecimalNumber
 -keep @interface pm.gnosis.heimdall.data.adapters.BigDecimalNumber
 
+-keep interface pm.gnosis.ethereum.rpc.models.** { *; }
+-keep class pm.gnosis.ethereum.rpc.models.** { *; }
```
Comment (biafra23) [inline app/proguard-rules.pro:77]: Just out of curiosity: Why do we need to keep this now and not before? What changed?

File: app/src/main/java/io/gnosis/safe/ui/StartActivity.kt [hunk 2]
```diff
@@ -416,8 +415,14 @@ class StartActivity : BaseActivity(), SafeOverviewNavigationHandler, AppStateLis
...
 
+    private fun onAppUpdated() {
+        showWhatsNew()
+        settingsHandler.showWhatsNew = false
+    }
+
     private fun showWhatsNew() {
-        navController.navigate(R.id.whatsNewDialog)
+        //TODO: uncomment if whats new screen should be shown
+        //navController.navigate(R.id.whatsNewDialog)
```
Comment (biafra23) [inline app/src/main/java/io/gnosis/safe/ui/StartActivity.kt:425]: Why do we need to do this manually?

File: build.gradle [hunk 3]
```diff
@@ -35,11 +35,11 @@ subprojects {
     afterEvaluate { project ->
         if (project.hasProperty("android")) {
             android {
-                compileSdkVersion 32
+                compileSdkVersion 33
 
                 defaultConfig {
                     minSdkVersion 21
-                    targetSdkVersion 32
+                    targetSdkVersion 33
```
Comment (biafra23) [inline build.gradle:42]: Do we have enough Android 13 devices to test on? My Pixel 3a doesn't run Android 13. Latest is Android 12. Maybe I need a new device to develop on?

File: gradle/wrapper/gradle-wrapper.properties [hunk 4]
```diff
@@ -4,4 +4,3 @@ distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
 distributionUrl=https\://services.gradle.org/distributions/gradle-7.3.3-all.zip
-x
```
Comment (biafra23) [inline gradle/wrapper/gradle-wrapper.properties:7]: Woot?

**Review by biafra23 — APPROVED** [review]
I would not go to targetSdkVersion 33 if we do not have to. 
If we do, then I need an Android 13 capable device for development.

File: app/proguard-rules.pro (hunk 1 above)
Comment (elgatovital) [reply app/proguard-rules.pro:77]: We have updated moshi and kotlin versions. My assumption is that either class generation syntax changed, which lead to this obfuscation error or that ens resolution was not working before in the first place.

File: app/src/main/java/io/gnosis/safe/ui/StartActivity.kt (hunk 2 above)
Comment (elgatovital) [reply app/src/main/java/io/gnosis/safe/ui/StartActivity.kt:425]: we do not have version based logic yet. What's new is always shown for the new version.

File: build.gradle (hunk 3 above)
Comment (elgatovital) [reply build.gradle:42]: you could use Android 13 emulator for now. Should be enough.

File: gradle/wrapper/gradle-wrapper.properties (hunk 4 above)
Comment (elgatovital) [reply gradle/wrapper/gradle-wrapper.properties:7]: that's a harmless typo. Cleanup is coming.

**Pull request merged**